java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -213,4 +224,90 @@ public interface BitmapBaseCommands {
      */
     CompletableFuture<Long> bitpos(
             String key, long bit, long start, long end, BitmapIndexType offsetType);
+
+    /**
+     * Reads or modifies the array of bits representing the string that is held at <code>key</code>
+     * based on the specified <code>subCommands</code>.
+     *
+     * @see <a href="https://redis.io/commands/bitfield/">redis.io</a> for details.
+     * @param key The key of the string.
+     * @param subCommands The <code>GET</code> subCommands to be performed on the binary value of the
+     *     string at <code>
+     *     key</code>.<br>
+     *     <ul>
+     *       <li>{@link BitFieldGet} gets the value in {@link Offset} or {@link OffsetMultiplier}
+     *           based on encoding being {@link SignedEncoding} or {@link UnsignedEncoding}.
+     *       <li>{@link BitFieldSet} sets the value in {@link Offset} or {@link OffsetMultiplier}
+     *           based on encoding being {@link SignedEncoding} or {@link UnsignedEncoding}.
+     *       <li>{@link BitFieldIncrby} increases or decreases the value in {@link Offset} or {@link
+     *           OffsetMultiplier} based on encoding being {@link SignedEncoding} or {@link
+     *           UnsignedEncoding}.
+     *       <li>{@link BitFieldOverflow} determines behaviour of {@link BitFieldSet} or {@link
+     *           BitFieldIncrby} when these operations result in under or overflows.
+     *     </ul>
+     *     <br>
+     *     Note:<br>
+     *     <ul>
+     *       <li>{@link Offset} and {@link OffsetMultiplier} must be greater than or equal to 0.
+     *       <li>{@link SignedEncoding} must be less than 64.
+     *       <li>{@link UnsignedEncoding} must be less than 65.
+     *     </ul>
```

Comment:
```suggestion
```
This could be deleted - it is already described in these classes' docs.
Your docs are pretty good and simple, there is no reason to duplicate them I think.